nVolve/Features/Content/ContentViewModel.swift

```diff
@@ -13,28 +13,13 @@ import SwiftUI
     var position: MapCameraPosition = .automatic
     var showEventInfo: Bool = false
     var showingFilters: Bool = false
-    var events: [InvolvedEvent] = []
-    // ':' is encoded as %3
-    // 'T' is the separator between time and date
+    var events: [EventModel] = []
+    var dataService = DataService()
```

Comment:
So just to make sure I'm understanding this PR. EventModel will standardize all the events we receive from the various sources (InvolvedEvent, TU Events, etc.) and that'll be combined together into the singular events array?